src/modules/10-common/hooks/useEventSourceListener.tsx

```diff
@@ -89,9 +91,17 @@ export function useEventSourceListener<T = unknown>({
         search: returnUrlParams(getLoginPageURL({ returnUrl: window.location.href }))
       })
     } else {
-      setSource(new EventSourcePolyfill(`${url}${getQueryParams(queryParams)}`, getRequestOptions()))
+      const { headers: defaultHeaders, ...rest } = getRequestOptions()
+      const headers = Object.assign({}, defaultHeaders, customHeaders)
```

Comment:
Nothing wrong here, but just curious why you went for `Object.assign` instead of just spreading the headers?

```suggestion
        const headers = { ...defaultHeaders, ...customHeaders }

Reply:
Old habit 😅 .
Is it performantly or behaviouraly any different?

Reply:
Honestly, if there is _any_ difference, it's WAY too minuscule to care about. Just been a while since I've seen this used and was curious if there was a specific reason you would chose one over the other.